kitchen/modules/gen-random/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub trait Trait: system::Trait {
 
 decl_storage! {
     trait Store for Module<T: Trait> as PGeneric {
-        Nonce get(nonce): u32;
+        Nonce get(fn nonce): u32;
```

Comment:
It doesn't seem to recognize this new syntax, doesn't compile locally for me :(

Reply:
This one doesn't compile because there are two `decl_stroage!`s. (lines 18 and 55). I agree it should be fixed, but I don't think it's in scope for this PR.